controllers/profiles/getProfileByName.js

```diff
@@ -49,7 +49,7 @@ module.exports = async (req, res) => {
     });
 
     const isFollowing = await sequelize.query(isFollowingQuery, {
-      replacements: {user_id_followed: targetUserId, user_id_follower: req.userId}
+      replacements: {user_id_follower: targetUserId, user_id_followed: req.userId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the order of `user_id_followed` and `user_id_follower` in the replacements object might affect the logic of the query. Please ensure that the SQL query `isFollowingQuery` is expecting these parameters in the new order. If not, this could lead to incorrect results or even a potential SQL error.
```diff
- replacements: {user_id_follower: targetUserId, user_id_followed: req.userId}
+ replacements: {user_id_followed: targetUserId, user_id_follower: req.userId}
```
If the SQL query is indeed expecting the parameters in the new order, then this change is correct and no further action is needed.


<!-- This is an auto-generated comment by OSS CodeRabbit -->